localstack/services/cloudformation/provider.py

```diff
@@ -161,30 +163,30 @@ def create_stack(self, context: RequestContext, request: CreateStackInput) -> Cr
 
         if (
             "CAPABILITY_AUTO_EXPAND" not in request.get("Capabilities", [])
-            and "Transform" in template
+            and "Transform" in template.keys()
```

Comment:
In that case the previous version would've been fine as well. Thought initially that this was a "x in string" operation. We'll also need to check the transform intrinsic functions in the future here though.